src/main/Service/display.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { AppConstants } from "../Constants/app-constant";
+import { DatePipe } from "@angular/common";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DisplayService {
```

Comment:
if this service is for calendar then we can rename it as CalendarDisplayService

Reply:
updated